src/jira/client/index.ts

```diff
@@ -153,13 +153,22 @@ async function getJiraClient(
 							}
 						}
 					),
-				delete: (gitHubInstallationId: string | number) =>
-					instance.delete(`/rest/devinfo/0.10/bulkByProperties`,
-						{
-							params: {
-								installationId: gitHubInstallationId
-							}
-						})
+				delete: async (gitHubInstallationId: string | number) =>
+					Promise.all([
+
+						// We are sending devinfo events with the property "installationId", so we delete by this property.
+						instance.delete(
+							`/rest/devinfo/0.10/bulkByProperties?installationId=${gitHubInstallationId}`
+						),
+
+						// We are sending builds and deployments with the property "gitHubInstallationId", so we delete by this property.
+						instance.delete(
+							`/rest/builds/0.1/bulkByProperties?gitHubInstallationId=${gitHubInstallationId}`
```

Comment:
Note that the property name `gitHubInstallationId` and not `installationId` like it is when we're deleting devinfo. That's because we're sending builds and deployments with the property `gitHubInstallationId`. See lines 244 and 268.